app/views/HomeView.coffee

```diff
@@ -96,7 +94,7 @@ module.exports = class HomeView extends RootView
     if anchorText
       @homePageEvent("Link: #{anchorText}", {}, ['Google Analytics'])
     else
-      @homePageEvent("Link:", {clicked: e?.currentTarget?.host or "unknown"}, [])
+      @homePageEvent("Link:", {clicked: e?.currentTarget?.host or "unknown"}, ['Google Analytics'])
```

Comment:
Why are we not using category 'Homepage' here?

Reply:
The method home page event attaches the category.